components/loading-indicator.js

```diff
@@ -0,0 +1,21 @@
+const React = require("react");
+const {connect} = require("react-redux");
+
+function LoadingIndicatorBase(props) {
+  const loadingClass = props.clientLoading ? 'is-loading' : '';
```

Comment:
<div class="loading-indicator {props.clientLoading && "is-loading"}">
  {props.clientLoading && props.children}
</div>

if(props.loading) {
  <d
}